crates/autopilot/src/arguments.rs

```diff
@@ -298,6 +303,51 @@ impl std::fmt::Display for Arguments {
     }
 }
 
+#[derive(clap::Parser, Clone)]
+pub struct FeePolicy {
+    /// How much of the order's price improvement over max(limit price,
+    /// best_bid) should be taken as a protocol fee.
+    #[clap(
+        long,
+        env,
+        default_value = None,
+        value_parser = shared::arguments::parse_percentage_factor
+    )]
+    pub price_improvement_factor: Option<f64>,
```

Comment:
Given that these are flattened, I think they should be prefixed with fee_policy or the like so that when we set or inspect the env variables of the pod it's clearer what `PRICE_IMPROVEMENT_FACTOR` relates to (cf. e.g. `ethrpc::Arguments`).

This goes for all of the variables below...

Reply:
prefixed